src/generate.js

```diff
@@ -23,20 +23,20 @@ export const generateCSS = (selector, styleTypes, stringHandlers,
         }
     });
 
-    return (
+    return [
         generateCSSRuleset(selector, declarations, stringHandlers,
-            useImportant) +
-        Object.keys(pseudoStyles).map(pseudoSelector => {
+            useImportant),
+        ...Object.keys(pseudoStyles).map(pseudoSelector => {
```

Comment:
Personally, I feel like this code could be refactored a bit to make it easier to read. Each item in this array should probably be assigned to a variable with a name that describes its purpose and then those could be put into the array as needed...